pkg/vfs/vfs_unix.go

```diff
@@ -290,3 +291,66 @@ func (v *VFS) Flock(ctx Context, ino Ino, fh uint64, owner uint64, typ uint32, b
 	}
 	return
 }
+
+func (v *VFS) Ioctl(ctx Context, ino Ino, cmd uint32, arg uint64, data []byte) (err syscall.Errno) {
```

Comment:
`arg` is not used

Reply:
Maybe other cmd will use the arg directly?